build.rs

```diff
@@ -10,7 +10,8 @@ fn main() {
         r"// AUTO-GENERATED CODE. DO NOT EDIT THIS FILE.",
         r#"pub const SETUP_TEMPLATE: &str = r##""#,
         cf_template,
-        r###""##;"###,
+        r###"
+"##;"###,
```

Comment:
This was changed to fix windows build. It wanted a return at the end of the block.